apps/docs/src/app/core/component-docs/list-byline/examples/list-byline-standard-example/list-byline-standard-example.component.html

```diff
@@ -0,0 +1,66 @@
+<h4>Standard Size</h4>
+<ul fd-list [byline]="true">
+    <li fd-list-item>
+        <span fd-list-thumbnail><fd-icon glyph="activate"></fd-icon></span>
+        <!-- for byline, fd-list-content is mandatory -->
+        <div fd-list-content>
+            <div fd-list-title>Title</div>
+            <div fd-list-byline>Byline Text</div>
+        </div>
+    </li>
+    <li fd-list-item>
+        <span fd-list-thumbnail><fd-icon glyph="employee"></fd-icon></span>
+        <div fd-list-content>
+            <div fd-list-title>List item with no byline</div>
+        </div>
+    </li>
+    <li fd-list-item>
+        <span fd-list-thumbnail><fd-icon glyph="calendar"></fd-icon></span>
+        <div fd-list-content>
+            <div fd-list-title>List Item With Two-Column Byline</div>
+            <div fd-list-byline [twoCol]="true">
+                <div fd-list-byline-left>First text item in Byline (Standard text item)</div>
+                <div fd-list-byline-right>Second text item in Byline (Can be semantic (Status) or not)</div>
+            </div>
+        </div>
+    </li>
+    <li fd-list-item>
+        <div fd-list-content>
+            <div fd-list-title>Text-Only List Item</div>
+            <div fd-list-byline>Byline Text (description)</div>
+        </div>
+    </li>
+</ul>
+
+<h4>Compact</h4>
```

Comment:
Do you need compact mode? I see there is no difference between compact and cozy modes

Reply:
The difference is the font-size of the title `fd-list__title`. In mobile it's 1rem, in compact is 0.875rem